local_groups/forms.py

```diff
@@ -50,7 +50,9 @@ class GroupCreateForm(forms.ModelForm):
 
     class Meta:
         model = Group
-        exclude = ('slug', 'signup_date', 'status', 'point')
+        exclude = (
+            'slug', 'signup_date', 'status', 'point', 'group_id', 'group_type'
```

Comment:
nice 👍

Reply:
I wonder if need to update any other forms too? Manage Group form?

Reply:
I wonder if would be better to have a whitelist of fields rather than exclude list? For case like this new field would be excluded automatically if we had field whitelist.